src/main/java/com/verisign/storm/metrics/reporters/graphite/GraphiteReporter.java

```diff
@@ -79,7 +81,14 @@ public GraphiteReporter() {
     graphiteSocketAddr = new InetSocketAddress(graphiteHost, graphitePort);
 
     serverFingerprint = graphiteSocketAddr.getAddress() + ":" + graphiteSocketAddr.getPort();
-    this.graphite = new Graphite(graphiteSocketAddr);
+
+    if (conf.containsKey(GRAPHITE_PROTOCOL_OPTION) && ((String)conf.get(GRAPHITE_PROTOCOL_OPTION)).equalsIgnoreCase("udp")) {
```

Comment:
I wasn't sure if it'd be better to keep backwards compat with existing configs and default to TCP, or to throw an exception if an appropriate config value was not defined